packages/components-angular/src/focus.directive.ts

```diff
@@ -1,12 +1,12 @@
-import { AfterViewInit, Directive, ElementRef } from '@angular/core'
+import { AfterViewChecked, Directive, ElementRef } from '@angular/core'
 
 @Directive({
-  selector: 'autofocus',
+  selector: '[balAutoFocus]',
```

Comment:
this is a breaking change or not?

Reply:
I think this never worked, we tried in our app and it did not so i thought to change name as well not to be as global attribute so i do not thing anyone has used it.